### PR TITLE
Simplified setup.

### PR DESCRIPTION
**Pull request by eldarm**
Sill TODO:
- readme
- push new images
- pass secrets for application and reader accounts.
- switch to our Google Cloud.

### CODE REVIEW

File: setup/setup.sh [hunk 1]
```diff
@@ -1,51 +1,111 @@
...
-
-# Done, we are running.
-echo *** Done, we are running.
\ No newline at end of file
+# Create .env file with secrets (***UPDATE BEFORE RUNNING!***):
+cat >.env << ENV
+# Setting secrets, update before running.
+MYSQL_ROOT_PASSWORD=secret
+MYSQL_APP_PASSWORD=secret
+MYSQL_READER_PASSWORD=secret
```
Comment (eugene-eko2000) [inline setup/setup.sh:17]: I would actually keep secrets in the .env file and create it separately. That would prevent modifying this file and an accidental secrets pushing on github.

**Review by eugene-eko2000 — CHANGES_REQUESTED** [review]

File: setup/setup.sh (hunk 1 above)
Comment (eldarm) [reply setup/setup.sh:16]: At first I did not think that's a reasonable concern, but it indeed looks better to set the passwords on a ephemeral file. Done.

File: setup/setup.sh [hunk 2]
```diff
@@ -2,16 +2,15 @@
...
-# 3. Update up.sh with real passwords before the first run!
 #
 # Give any arg to skip the installation, e.g. "./setup.sh local"
 
-# Create .env file with secrets (***UPDATE BEFORE RUNNING!***):
+# Create necessary files.
+
 cat >.env << ENV
-# Setting secrets, update before running.
+# Setting secrets, please, update with real passwords and save/exit editor.
```
Comment (eugene-eko2000) [inline setup/setup.sh:13]: This comment isn't fully clear. I'm seeing the vim .env which looks correct. This comment and variables definition in this file doesn't look relevant to it.

File: setup/setup.sh [hunk 3]
```diff
@@ -58,54 +57,63 @@ volumes:
...
-    installDocker
-
-    # Pull images:
-    docker pull ibnazer/cleanappserver:1.6
-    docker pull ibnazer/cleanappdb:1.6
-
-    # Start our docker images.
-    ./up.sh
-    echo "*** We are running."
+# Set passwords. On the target machine change to yhour favorite etxt editor:
```
[the remark this answers is not in the document]
Comment (eldarm) [reply setup/setup.sh]: Fixed.

File: setup/setup.sh (hunk 2 above)
Comment (eldarm) [reply setup/setup.sh:13]: Expanded. This comment does not belong to setup.sh, it's the content of generated .env file. This way this is at least clear, other ways to insert such instruction into .env file will be much more confusing.

**Review by eugene-eko2000 — APPROVED** [review]
